### PR TITLE
[Desktop] App & template update to itwins api

### PR DESCRIPTION
**Pull request by Jake-Screen**
- Updating the desktop-viewer-template and desktop-test-app to use the 1.0.0 versions of the imodel-browser-react package.
- Switching from using the projects api to the new itwins api.

### CODE REVIEW

File: packages/apps/desktop-viewer-test/src/frontend/index.tsx [hunk 1]
```diff
@@ -19,19 +19,14 @@ const viewerFrontendMain = async () => {
   Logger.setLevelDefault(LogLevel.Trace);
   Logger.setLevel(AppLoggerCategory.Frontend, LogLevel.Info);
 
-  // TODO add theme toggle
-  document.documentElement.classList.add(`iui-theme-dark`);
```
Comment (aruniverse) [inline packages/apps/desktop-viewer-test/src/frontend/index.tsx:23]: what happened to this?

File: packages/templates/cra-template-desktop-viewer/template/README.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ ITWIN_VIEWER_CLIENT_ID="native-xxxxxxxx"
   - **Visualization**: `imodelaccess:read`
   - **iModels**: `imodels:read`
   - **Reality Data**: `realitydata:read`
-  - **Projects**: `projects:read`
+  - **itwins**: `itwins:read`
```
Comment (aruniverse) [inline packages/templates/cra-template-desktop-viewer/template/README.md:20]: there is no `itwins` association

Comment (aruniverse) [reply packages/templates/cra-template-desktop-viewer/template/README.md:20]: it's administration: https://developer.bentley.com/api-groups/administration/

Comment (aruniverse) [reply packages/templates/cra-template-desktop-viewer/template/README.md:20]: the `scope` is `itwins:read`
but the `association` is `Administration`

Comment (Jake-Screen) [reply packages/templates/cra-template-desktop-viewer/template/README.md]: changed

File: common/changes/@itwin/viewer-react/jake-itwins-api_2023-07-25-14-54.json
Comment (aruniverse) [inline common/changes/@itwin/viewer-react/jake-itwins-api_2023-07-25-14-54.json:1]: this file is irrelevant now, you can delete it

but also, if these changes did exist, why would the `type` be none?

**Comment by aruniverse** [comment]
there seems to be an extra highlight on the active tab, is that new? 
image

**Comment by Jake-Screen** [comment]
> there seems to be an extra highlight on the active tab, is that new? image

This was not present before, changed to previous appearance

File: common/changes/@itwin/viewer-react/jake-itwins-api_2023-07-25-14-54.json
Comment (Jake-Screen) [reply common/changes/@itwin/viewer-react/jake-itwins-api_2023-07-25-14-54.json:1]: deleted. Should have been minor i believe due to changing the version of a direct dependency

**Review by aruniverse — APPROVED** [review]

**Pull request merged**